index.html

```diff
@@ -14,21 +14,24 @@
 			<div class="container">
 				<div class="header__content flex-row align-center justify-between">
 					<a href="#" class="header__link color-primary">My Logo</a>
-					<img
-						class="header__toggle"
-						src="images/icons/burguer.svg"
-						alt="Menu icon on the mobile version"
-					/>
-					<nav class="header__nav">
-						<ul class="header__list flex-row align-center">
-							<li class="header__list--item">
-								<a class="color-info" href="#portfolio">Portfolio</a>
+					<button href="#" id="header__button">
+						<img
+							id="header__button--image"
+							class="header__toggle"
+							src="images/icons/burguer.svg"
+							alt="Menu icon on the mobile version"
+						/>
+					</button>
+					<nav class="header__nav header__nav--inactive" id="header__nav">
+						<ul class="header__list header__list--inactive flex-row align-center">
+							<li>
+								<a class="header__list--item" href="#portfolio">Portfolio</a>
 							</li>
-							<li class="header__list--item">
-								<a class="color-info" href="#about">About</a>
+							<li>
+								<a class="header__list--item" href="#about">About</a>
 							</li>
-							<li class="header__list--item">
-								<a class="color-info" href="#contact">Contact</a>
+							<li>
+								<a class="header__list--item" href="#contact">Contact</a>
```

Comment:
- [ ] Kindly ensure that you design sticks as much possible to the one of your chosen Figma template. Right now, there are some differences on `fonts(weight, )`, `broken images`, ...

| Figma design | Your Design |
| ------------------- | ------------------- |
| ![Screenshot from 2023-06-20 01-03-37](https://github.com/Rysth/Portfolio-Project/assets/83042742/f3768589-b057-46f9-b6d0-ed02ccc6d2ae) | ![Screenshot from 2023-06-20 01-02-39](https://github.com/Rysth/Portfolio-Project/assets/83042742/3c40bd4c-c7aa-4333-a1d9-bcfe808a3edd) |

Reply:
Thanks, @tresorsawasawa!

### 🏅 Fixeds Implemented

List of things that were fixed based on the last review:

- ✅ Added a new `eslintrc.json` and name replaced based on the comment from your review.
- ✅ Fixed the `icons` to open/close the mobile menu.
- ✅ Matched as close as possible the `background color` based on the Figma Template.

_John Palacios_
_Microverse Student_